components/expand-collapse/test/expand-collapse.visual-diff.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script type="module">
+		import '../../typography/typography.js';
+		import '../expand-collapse.js';
+	</script>
+	<title>d2l-expand-collapse</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<meta charset="UTF-8">
+	<style>
+		ul {
+			border: 4px solid green;
+		}
+		d2l-expand-collapse {
+			border: 4px solid blue;
+		}
+	</style>
+</head>
+
+<body class="d2l-typography">
+	<div class="visual-diff">
+		<d2l-expand-collapse id="collapsed">
+			<ul>
+				<li>Coffee</li>
+				<li>Tea</li>
+				<li>Milk</li>
+			</ul>
+		</d2l-expand-collapse>
```

Comment:
Interesting that the blue border still appears even when it's collapsed. It makes sense given it the internal container that gets a `display: none` on it, but as a consumer of this I wouldn't have expected it. I can't think of a super clean way to fix this without reflecting that `_state` property up to the host. Hmm...

Reply:
I think having the border still show up will work well. It aligns with FACE's use of expand collapse where they have a line where the expand collapse section is when collapsed. Implementing it like this means it can be done with an attribute selector: 
![Capture](https://user-images.githubusercontent.com/11379933/82069651-a0ce5800-96a1-11ea-9483-3fecc7ca2399.PNG)

It will also make it a lot easier to use with `d2l-more-less` since that component doesn't fully collapse by default, it leaves 4 ems visibile.